projects/sound-open-firmware/Dockerfile

```diff
@@ -68,6 +68,9 @@ RUN cd $SRC; touch empty.c; gcc -m32 -c empty.c; ar rs /usr/lib32/libstdc++.a em
 # If a user is mounting their local source directory for debugging
 # then we need to tell git to chill out and just take the repo. If we
 # don't west fails with bogus error messages.
-RUN git config --global --add safe.directory /src/sof_workspace/zephyr
+RUN git config --global --add safe.directory $SRC/sof_workspace/zephyr
+
+# This is a HACK to get around the fact that we don't have python 3.10 in oss-fuzz yet
```

Comment:
Maybe it's bad etiquette vs. the folks managing the backend, but you could install a python interpreter into the image.  Not sure how often the images are regenerated or how much people care about storage for the extra stuff.

Reply:
There is an [inflight PR](https://github.com/google/oss-fuzz/pull/12027) to fix this already (and its not trivial, otherwise I would have done that). Also image generation is ~1/day.

Reply:
Probably a non-issue if that gets merged soon, but I meant that building python from source is actually really easy.  I haven't done it in a while but just checked and this works:

```bash
git clone https://github.com/python/cpython
cd cpython
./configure --enable-optimizations --prefix=$SOME_LOCAL_INSTALL_DIR
make -j$(nproc)
make install
```

Would be best to check out the 3.14 tag or whatever, obviously.  And would need to be followed with a pip3 (no venv needed as it's a local install) to update, etc...

Reply:
switched to pulling deadsnakes to upgrade